<launch>
	<node pkg="joy" name="joystick" type="joy_node" machine="shoreside"
		unless="$(env IS_REMOTE)">
		<param name="dev" value="$(env JOYSTICK)" />
		<param name="coalesce_interval" value="0.1" />
		<param name="autorepeat_rate" value="10" />
		<param name="deadzone" value="0.05" />
	</node>

	<group if="$(env IS_REMOTE)">
		<node pkg="cart2" type="radio_node" name="radio_modem">
			<param name="PortName" value="/dev/ttyUSB0" />
			<param name="Baud" value="9600" />
			<remap from="joy_out" to="joy" />
		</node>
	</group>
</launch>
